canvas_account_admin_tools/views.py

```diff
@@ -55,40 +55,27 @@ def lti_launch(request):
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
-@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
+@lti_permission_required(PERMISSIONS['canvas_account_admin_tools'])
```

Comment:
Right, we would just have to use the permission name here. There would be no verification that it would exist in the TOOL_PERMISSIONS list; we'd just have to make sure they matched up e.g. by eye.